deps/redis/listpack_malloc.h

```diff
@@ -39,12 +39,24 @@
 #ifndef LISTPACK_ALLOC_H
 #define LISTPACK_ALLOC_H
 //#include "zmalloc.h"
+
+#ifdef __APPLE__
+#include <malloc/malloc.h>
+#else
 #include "malloc.h"
+#endif
+
 /* We use zmalloc_usable/zrealloc_usable instead of zmalloc/zrealloc
  * to ensure the safe invocation of 'zmalloc_usable_size().
  * See comment in zmalloc_usable_size(). */
```

Comment:
This comment is no longer relevant. Please remove it.

Reply:
Will do!